### PR TITLE
SoftMaker.FreePDF.2022 version 3.0.8

### PR DESCRIPTION
**Pull request by borque**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90549)

### CODE REVIEW

**Comment by borque** [comment]
Hello @mdanish-kh , @russellbanks !
We were discussing whether th previous version of FreePDF (PR #89292 ) should be called 2019 or not. The latest update to 3.0.8 officially changed the name to FreePDF 2022 (even on their homepage) so I created the new manifest again with the **.2022** name and wanted to aks for your opinion, whether it is okay this time or whether I should change it back to the scheme without the appended year and simply increase the version to 3.0.8?
If we decide to use the .2022, should we then change the old manifest to .2019 because SoftMaker sometimes called it so and to be congruent with the current version or leave it the way it is?

**Comment by borque** [comment]
@msftbot : make sure @russellbanks or @mdanish-kh has a chance to review before you merge

**Comment by mdanish-kh** [comment]
Yep, looks like the publisher made some noticeable changes to their website and provided a new installer for the 2022 version. The changes in this PR look good to me. However, I'm still not comfortable changing the previous manifest to `.2019` as the reasons described in https://github.com/microsoft/winget-pkgs/pull/89292 still stand.

Another argument to compliment previous ones in https://github.com/microsoft/winget-pkgs/pull/89292 is that the new installer installs to `C:\Program Files (x86)\SoftMaker\FreePDF 2022` while the old one installs to `C:\Program Files (x86)\SoftMaker\FreePDF`. Unless the behavior is changed by the publisher, I still don't see a convincing reason to change the previous manifest.

Will wait for russellbanks's take on it.

File: manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SoftMaker.FreePDF.2022
+PackageVersion: 3.0.8
+InstallerType: inno
```
Comment (mdanish-kh) [inline manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.installer.yaml:6]:
```suggestion
InstallerType: inno
AppsAndFeaturesEntries:
- DisplayVersion: '3.08'
```
The publisher markets the version as `3.0.8` but the version reported to Registry (visible in Control Panel) is `3.08`

File: manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SoftMaker.FreePDF.2022
+PackageVersion: 3.0.8
+PackageLocale: en-US
+Publisher: SoftMaker Software GmbH
+PackageName: FreePDF 2022
+License: Proprietary
+Copyright: SoftMaker Software GmbH
```
Comment (mdanish-kh) [inline manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.locale.en-US.yaml:10]:
```suggestion
Copyright: Copyright (c) 2022 SoftMaker
```
Got it from the package's website

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by russellbanks** [comment]
> Another argument to compliment previous ones in #89292 is that the new installer installs to `C:\Program Files (x86)\SoftMaker\FreePDF 2022` while the old one installs to `C:\Program Files (x86)\SoftMaker\FreePDF`. Unless the behavior is changed by the publisher, I still don't see a convincing reason to change the previous manifest.
> 
> Will wait for russellbanks's take on it.

I agree with you again on this - Installing to a directory with/without the year in it further shows the publisher's intention for that version.

**Review by mdanish-kh — APPROVED** [review]

**Comment by russellbanks** [comment]
@mdanish-kh, I think the InstallerURL architecture here should be `x86`/`x64 & x86` as neutral covers all architectures (ARM, etc). Task Manager reports it as x86.
WindowsSandboxClient_yKQLFTmURO

**Comment by mdanish-kh** [comment]
Oh right! I thought about it and let it go since I assumed the validation pipelines might report if something was off. But Esco once mentioned in a PR https://github.com/microsoft/winget-pkgs/pull/56375#discussion_r839609547 that we've moved away from `neutral`, so I think this should be updated.

**Pull request merged**